pkg/profile/issuer/profile.go

```diff
@@ -45,7 +46,7 @@ type ProfileData struct {
 	OIDCClientParams            *OIDCClientParams `json:"oidcParams,omitempty"`
 	CredentialScopes            []string          `json:"credScopes,omitempty"`
 	LinkedWalletURL             string            `json:"linkedWallet,omitempty"`
-	// Todo #issue Add credential manifest issuer object
+	CredentialManifestIssuer    cm.Issuer         `json:"issuer,omitempty"`
```

Comment:
Can't we use `Name` from [line 36](https://github.com/trustbloc/adapter/pull/574/files#diff-545db50b97be2ca12b6616f131476ab403b1cc28c7a3de1c9fa1400886c1950cR36) rather a new variable ?

Reply:
I was wondering same, profile name & issuer names are same? remember `issuer.Name` name gets displayed in.wallet UI.

Reply:
It is - same is displayed in DIDComm CHAPI screens as well during sharing (passed as part of credential.issuer field).

Reply:
@talwinder50 remove `cm.Issuer` and use 

- Name - already there
- IssuerID - put comment as it is going to be used as issuer ID in credential manifests  being issued
- Style - style object from aries credential manifest package, add go comments to explain it.

Reply:
Updated

Reply:
@talwinder50 I don't see any updates